### DISCUSSION TITLE
Welcome to xArmServoController Discussions!

### DISCUSSION THREAD

**Discussion by ccourson** [ccourson/Hiwonder-xArm1S#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JerrySlavec** [comment]
Hello xArmServoController community!
My name is Jerry Slavec and I've recently acquired an xArm 1S robotic device.  I have done a fair amount of programming with various Arduino's, but this will be my first attempt to control an xArm with an Arduino.  I think the xArmServoController is the only library available for the Arduino, although it doesn't appear in the IDE 2.3.2 Library selection screen.  Can you direct me on how I can download and use this library?  Thank you very much!
